app/src/main/java/com/keylesspalace/tusky/di/StorageModule.kt

```diff
@@ -65,8 +66,11 @@ object StorageModule {
                 AppDatabase.MIGRATION_41_42, AppDatabase.MIGRATION_42_43, AppDatabase.MIGRATION_43_44,
                 AppDatabase.MIGRATION_44_45, AppDatabase.MIGRATION_45_46, AppDatabase.MIGRATION_46_47,
                 AppDatabase.MIGRATION_47_48, AppDatabase.MIGRATION_52_53, AppDatabase.MIGRATION_54_56,
-                AppDatabase.MIGRATION_58_60
+                AppDatabase.MIGRATION_58_60, AppDatabase.MIGRATION_58_60
             )
             .build()
     }
+
+    @Provides
+    fun providesAccountDao(db: AppDatabase): AccountDao = db.accountDao()
```

Comment:
But `AccountDao` isn't injected anywhere? Either remove this again or actually inject it somewhere.

Reply:
Good catch. This is an artifact of the merge resolution I did because `AccountDao` is now being retrieved from the DB object instead of injected. Seems simplest to remove the provider for now since we don't really need to mock it in the unit tests, I am mocking the account manager instead.